pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -211,6 +219,12 @@ func (k *Deployer) Deploy(ctx context.Context, out io.Writer, builds []graph.Art
 	endTrace()
 
 	childCtx, endTrace := instrumentation.StartTrace(ctx, "Deploy_renderManifests")
+	if err := k.imageLoader.LoadImages(childCtx, out, k.localImages, k.originalImages, builds); err != nil {
```

Comment:
super nit: add a trace for `Deploy_LoadImages`

Reply:
copy paste error :)